pkg/repositories/gormimpl/execution_repo.go

```diff
@@ -110,6 +110,23 @@ func (r *ExecutionRepo) List(ctx context.Context, input interfaces.ListResourceI
 	}, nil
 }
 
+func (r *ExecutionRepo) Count(ctx context.Context, input interfaces.CountResourceInput) (int64, error) {
+	var err error
+	tx := r.db.Model(&models.Execution{})
+	tx, err = applyScopedFilters(tx, input.InlineFilters, input.MapFilters)
```

Comment:
for completeness can we add the calls to [JoinTableEntities](https://github.com/flyteorg/flyteadmin/blob/master/pkg/repositories/gormimpl/execution_repo.go#L78,L90) so we can support cross table filters